website/content/docs/security/acl/acl-system.mdx

```diff
@@ -2,72 +2,52 @@
+The Consul Access Control List (ACL) system authenticates requests and authorizes
+access to resources. It is used by the UI, API, CLI, service-to-service
+communication, and agent-to-agent communication.
 
-Consul provides an optional Access Control List (ACL) system which can be used to control access to data and APIs.
-The ACL is [Capability-based](https://en.wikipedia.org/wiki/Capability-based_security), relying on tokens which
-are associated with policies to determine which fine grained rules can be applied. Consul's capability based
-ACL system is very similar to the design of [AWS IAM](https://aws.amazon.com/iam/).
+This page will introduce you to all of the core concepts of the ACL system. For a more
+step-to-step guide see [Bootstrap and Explore ACLs] for getting started with ACLs and
+[Secure Consul with ACLs] for configuring ACLs in a production environment.
 
-To learn how to setup the ACL system on an existing Consul datacenter, use the [Bootstrapping The ACL System tutorial](https://learn.hashicorp.com/tutorials/consul/access-control-setup?utm_source=consul.io&utm_medium=docs).
+[Bootstrap and Explore ACLs]: https://learn.hashicorp.com/tutorials/consul/access-control-setup?utm_source=consul.io&utm_medium=docs
+[Secure Consul with ACLs]: https://learn.hashicorp.com/tutorials/consul/access-control-setup-production?utm_source=consul.io&utm_medium=docs
 
-## ACL System Overview
+See allso the [ACL API reference](/api-docs/acl), [ACL CLI reference](/commands/acl), and
+[Troubleshoot the ACL System](https://learn.hashicorp.com/tutorials/consul/access-control-troubleshoot).
 
-The ACL system is designed to be easy to use and fast to enforce while providing administrative insight.
-The diagram below shows the relationships between most of the components of the ACL system:
 
-![ACL system component relationships](/img/acl-token-policy-rule-relationship.png)
-
-At the highest level, there are two major components to the ACL system:
-
-- **ACL Policies** - Policies allow the grouping of a set of rules into a logical unit that can be reused and linked with
-  many tokens.
+## Overview
 
-- **ACL Tokens** - Requests to Consul are authorized by using bearer token. Each ACL token has a public
-  Accessor ID which is used to name a token, and a Secret ID which is used as the bearer token used to
-  make requests to Consul.
+The diagram below shows the relationships between the core components of the ACL system:
 
-For many scenarios policies and tokens are sufficient, but more advanced setups
-may benefit from additional components in the ACL system:
-
-- **ACL Roles** - Roles allow for the grouping of a set of policies and service
-  identities into a reusable higher-level entity that can be applied to many
-  tokens. (Added in Consul 1.5.0)
+![ACL system component relationships](/img/acl-token-policy-rule-relationship.png)
```

Comment:
Maybe this diagram should be moved to the end of the "Tokens" section, what do you think?

That way it comes after the description of a token which talks about policies.